Update nginx.md

jeremiah opened the pull request:
Minor comments regarding nginx configuration.

jeremiah committed:
Update nginx.md

Minor comments regarding nginx configuration.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/stalwartlabs/website?pullRequest=57) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/stalwartlabs/website?pullRequest=57) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/stalwartlabs/website?pullRequest=57) it.</sub>

stappersg commented on docs/server/reverse-proxy/nginx.md:
````diff
@@ -53,3 +53,13 @@ stream {
     }
 }
 ```
+You'll need to stop Stalwart mail so that nginx can bind to the appropriate ports. Then restart ngxin with the Stream configuration and restart afterwards Stalwart.
````
Please _line wrap_ at 72 characters or another amount of character less as 81.

jeremiah replied:
Will do.

jeremiah committed:
Update nginx.md

Added some text for greater technical clarity. Added name of stream module package in Debian based systems.

jeremiah committed:
Update nginx.md

Add in a bit more clarifying text about alternative ports and some technical details for Debian based systems.

mdecimus commented:
Thanks @jeremiah. The NGINX page was rewritten for v0.16; the surrounding section this PR extends has changed. Closing as superseded.
